broker/broker.go

```diff
@@ -6,11 +6,10 @@ import (
 	"errors"
 	"fmt"
 	"net/url"
-	"strings"
 
 	"code.cloudfoundry.org/lager/v3"
 	"github.com/aws/aws-sdk-go/service/s3"
-	"github.com/cloudfoundry-community/go-cfclient"
+	cfclient "github.com/cloudfoundry-community/go-cfclient/v3/client"
```

Comment:
There's no harm in this, but it's probably better to import it as just `"github.com/cloudfoundry-community/go-cfclient/v3/client"`, right? 

Otherwise we run into confusion with the `S3Broker.cfClient` property and this imported `cfclient` package name. The casing is different, but it's still somewhat confusing. I could be missing something here

Reply:
I think you're right that the names look too similar, but I'm going to solve it a different way: Change the field name to `cf` and keep the import alias. I think the alias should stay because it makes lines like this more readable: 

```go
type S3Broker struct {
	provider                     provider.Provider
	insecureSkipVerify           bool
	// ...
	cf                           *cfclient.Client // this one
}
```
I like that you can see at a glance that it's a Cloud Foundry client, not a `client.Client`. As a bonus, I think renaming the field to `cf` is more idiomatic Go naming anyway — it's terse and unambiguous. (There's nothing cf-related it could be referring to besides the cf client.)

Reply:
I like it